doc/1/controllers/index/get-auto-refresh/index.md

```diff
@@ -0,0 +1,38 @@
+---
+code: true
+type: page
+title: GetAutoRefreshAsync
+description: Returns the status of autorefresh flag.
+---
+
+# GetAutoRefreshAsync
+
+The getAutoRefresh action returns the current autorefresh status for the index.
+
+Each index has an autorefresh flag.  
+When set to true, each write request triggers a [refresh](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-refresh.html) action in Elasticsearch.  
```

Comment:
Do not link to the "current" ES documentation, which currently points to the ES7 documentation.
Use specifically the v5 version:

```suggestion
When set to true, each write request triggers a [refresh](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/docs-refresh.html) action in Elasticsearch.  
```